Tasks/VsTest/taskinputparser.ts

```diff
@@ -186,7 +187,12 @@ function initTestConfigurations(testConfiguration: models.TestConfigurations) {
 
     testConfiguration.taskInstanceIdentifier = uuid.v1();
 
-    versionFinder.getVsTestRunnerDetails(testConfiguration);
+    try {
+        versionFinder.getVsTestRunnerDetails(testConfiguration);
+    } catch (error) {
+        utils.Helper.publishEventToCi(AreaCodes.SPECIFIEDVSVERSIONNOTFOUND, error.message, 1039, true);
+        tl.setResult(tl.TaskResult.Failed, error);
```

Comment:
Dont set result here. Throw here and catch where this function is called (i.e. main vstest.ts). We should not set result at any more places now

Reply:
okay